common/browser.go

```diff
@@ -361,6 +361,44 @@ func (b *Browser) isAttachedPageValid(ev *target.EventAttachedToTarget, browserC
 	return true
 }
 
+// isPageAttachmentErrorIgnorable returns true if the error is ignorable.
+func (b *Browser) isPageAttachmentErrorIgnorable(ev *target.EventAttachedToTarget, session *Session, err error) bool {
+	targetPage := ev.TargetInfo
+
+	// If we're no longer connected to browser, then ignore WebSocket errors.
+	// This can happen when the browser is closed while the page is being attached.
+	var (
+		isRunning = atomic.LoadInt64(&b.state) == BrowserStateOpen && b.IsConnected() // b.conn.isConnected()
+		wsErr     *websocket.CloseError
+	)
+	if !errors.As(err, &wsErr) && !isRunning {
+		// If we're no longer connected to browser, then ignore WebSocket errors
+		b.logger.Debugf("Browser:onAttachedToTarget:return", "sid:%v tid:%v pageType:%s websocket err:%v",
```

Comment:
```suggestion
		b.logger.Debugf("Browser: isPageAttachmentErrorIgnorable:return", "sid:%v tid:%v pageType:%s websocket err:%v",
```

Same for the others in this method.

Reply:
I missed this, thanks. See: 745cf5499.